src/q2g-ext-listboxExtension.less

```diff
@@ -0,0 +1,36 @@
+.root-container {
+    height:100%;
+    width:100%;
+
+    .editMode {
+        position: absolute;
+        width: 100%;
+        height: 100%;
+        z-index: 1000;
+        top: 0;
+    }
+}
+
+
+
+
+
+.q2g-selection {
```

Comment:
@thomashaenig correct namespace?? I think that have to be something with listbox